include/oneapi/tbb/task_group.h

```diff
@@ -211,7 +212,8 @@ class task_group_context : no_copy {
         locked,
         isolated,
         bound,
-        dead
+        dead,
+        proxy = std::numeric_limits<std::underlying_type<lifetime_state>::type>::max()
```

Comment:
As a suggestion:
The underlying type is specified for lifetime_state(std::uint8_t), so:
```suggestion
        proxy = std::uint8_t(-1)
```